tracker.js

```diff
@@ -1,24 +1,86 @@
-var BtczBalance1 = 0;
-var BtczBalance2 = 0;
 var BtczValue = 0;
+var LtcValue = 0;
+var EthValue = 0;
+var ZecValue = 0;
+
 var startDate = new Date("2018-01-03T19:00:00Z");
 
+var BtczBalance1 = 0;
+var BtcBalance1 = 0;
+var LtcBalance1 = 0;
+var ZecBalance1 = 0;
+var EthBalance1 = 0;
+
+var BtczBalance2 = 0;
+var BtcBalance2 = 0;
+var LtcBalance2 = 0;
+var ZecBalance2 = 0;
+var EthBalance2 = 0;
+
+
 function updatePage(){
     var currentDate = new Date().getTime();
     var hours = Number(Math.round((currentDate-startDate)/3600000));
     var totalDonations = Number(Math.round(BtczBalance1+BtczBalance2));
     var hourlyAverage = Number(Math.round(totalDonations/hours));
-    var BtczBalance1_toBtc = (BtczBalance1*BtczValue);
-    var BtczBalance2_toBtc = (BtczBalance2*BtczValue);
+
+
     document.getElementById('statsTxt').innerHTML = "<b>" + totalDonations + " BTCZ</b> raised in <b>" + hours + " hours</b>, average: <b>" + hourlyAverage + " BTCZ/h</b>";
-    document.getElementById('btczBalance1').innerHTML = BtczBalance1.toFixed(0) + " BTCZ";
+
+	
+	var BtczBalance1_toBtc = (BtczBalance1*BtczValue);
+	document.getElementById('btczBalance1').innerHTML = BtczBalance1.toFixed(0) + " BTCZ";
```

Comment:
Good ol' `document.getElementById`.

Reply:
old school - just following the scheme, respecting the previous coder ;)

Reply:
haha, yeah. Good stuff. Taking me back to the good ol' days.